src/topology/metric_space/basic.lean

```diff
@@ -10,6 +10,7 @@ import topology.algebra.order.compact
 import topology.metric_space.emetric_space
 import topology.bornology.constructions
 import topology.uniform_space.complete_separated
+import topology.uniform_space.equicontinuity
```

Comment:
This file `metric_space.basic` is already too big. What about creating a new file instead?

Reply:
Works for me, I'll do that.